.github/workflows/jmh-benchmark.yml

```diff
@@ -0,0 +1,79 @@
+name: Run JMH Benchmarks for Pull Request
+
+on:
+  issue_comment:
+    types: [created]
+
+
+jobs:
+  setup:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/benchmark') && github.repository == 'uber/NullAway'
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set branch name
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        PR_NUMBER=${{ github.event.issue.number }}
+        PR_DETAILS=$(curl -s -H "Authorization: token $GH_TOKEN" "https://api.github.com/repos/${{ github.repository }}/pulls/$PR_NUMBER")
+        REPO_NAME=$(echo "$PR_DETAILS" | jq -r .head.repo.full_name)
+        BRANCH_NAME=$(echo "$PR_DETAILS" | jq -r .head.ref)
+        echo "REPO_NAME=$REPO_NAME" >> $GITHUB_ENV
+        echo "BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_ENV
```

Comment:
Is there any way to pull this sequence of commands into a separate shell script?  Again, it will be easier to review, document, etc.

Reply:
Fixed